docs/hydra/guides/graceful-token-refresh.mdx

```diff
@@ -55,10 +55,21 @@ with new tokens for each request without immediate invalidation of the original
 
 ## Example behavior with grace period
 
-- **Using the refresh token within the grace period**: If a refresh token is used twice within the configured grace period (for
+When the user calls `/oauth2/auth` and performs login and consent, the OAuth2 server issues an access token and a refresh token.
+These tokens and all subsequent tokens issued within the grace period are part of the same consent request.
```

Comment:
The phrase "consent request" is, I think, a Hydra implementation artifact.

Maybe some day we can clarify the concepts both in documenation and in code.

It's a difficult thing to wrap your head around. This is just a side remark.

This PR is 👍

Reply:
Yeah, I actually struggled quite hard in trying to find the right phrase. Is it login request? consent request? token chain? consent flow? authorized consent id? Haven't found a good solution yet.